pkg/slo-controller/noderesource/plugins/gpudeviceresource/plugin.go

```diff
@@ -151,6 +151,17 @@ func (p *Plugin) Calculate(_ *configuration.ColocationStrategy, node *corev1.Nod
 		return p.resetGPUNodeResource()
 	}
 
+	existsGPU := false
+	for _, d := range device.Spec.Devices {
+		if d.Type == schedulingv1alpha1.GPU && d.Health {
+			existsGPU = true
+		}
+	}
+	if !existsGPU {
+		klog.V(5).InfoS("device not found, reset gpu resources on node")
```

Comment:
How about `device not found` -> `gpu not found in device`?
And the log of the node-level calculation should print with the node name.